### PR TITLE
Add Supporting Information video on editing via GitHub

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/139

### CODE REVIEW

**Comment by dhimmel** [comment]
HTML looks like:

>  image

PDF looks like:

> image

I'll remove "Your browser cannot display this video.", which while useful for browsers, is probably not something we want in the PDF or DOCX output.

File: content/02.main-text.md [hunk 1]
```diff
@@ -440,3 +440,24 @@ Source code for Manubot resides in the following repositories:
...
+>
+  <source
+    src="https://ndownloader.figshare.com/files/14794016/preview/14794016/video_preview.mp4"
+    type="video/mp4"
+  >
+  Your browser cannot display this video.
+</video>
+**S1 Video: Editing a manuscript on GitHub.**
+This screen recording demonstrates how to propose edits to a Manubot manuscript via GitHub.
+In the video [@doi:10.6084/m9.figshare.7946192.v1], a contributor creates a [pull request](https://github.com/manubot/try-manubot/pull/2) to add a sentence to the try-manubot manuscript.
```
Comment (dhimmel) [inline content/02.main-text.md:460]: @agitter should I capitalize "contributor" like Contributor to match discussion of Figure 1?

Comment (agitter) [reply content/02.main-text.md:460]: I think lower case is okay here.  I was treating Contributor, Participant, Maintainer in Figure 1 as placeholder proper nouns like Alice, Bob, etc.  In other text I've used `contributor`.

**Comment by slochower** [comment]
Thanks @dhimmel

**Comment by dhimmel** [comment]
@agitter want to review for mergeability?

**Review by agitter — APPROVED** [review]
Looks good to me

File: content/02.main-text.md [hunk 2]
```diff
@@ -440,3 +440,23 @@ Source code for Manubot resides in the following repositories:
 
 + <https://github.com/manubot/manubot> ([GitLab mirror](https://gitlab.com/manubot/manubot), archived at [`swh:1:dir:1eb3b7e6d7e21239fedccf25a186af622ee6912e`](https://archive.softwareheritage.org/swh:1:dir:1eb3b7e6d7e21239fedccf25a186af622ee6912e;origin=https://github.com/manubot/manubot.git/), packaged on [PyPI](https://pypi.org/project/manubot/))
 + <https://github.com/manubot/rootstock> ([GitLab mirror](https://gitlab.com/manubot/rootstock), archived at [`swh:1:dir:cfc3af2e8e1a0d9b639fdab0943731e608910731`](https://archive.softwareheritage.org/swh:1:dir:cfc3af2e8e1a0d9b639fdab0943731e608910731;origin=https://github.com/manubot/rootstock.git/)).
+
+## Supporting Information
+
+<div id="vid:usage">
```
Comment (dhimmel) [inline content/02.main-text.md:446]: Noting that the recently released Pandoc 2.7.3 may have allow a more markdown-like syntax for video embeds. See https://twitter.com/dhimmel/status/1148978224996556800.

**Pull request merged**